<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BluetoothAssisWidget</class>
 <widget class="QWidget" name="BluetoothAssisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>蓝牙设置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>设备:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cb_devices">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="btn_refresh">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/icon/res/refresh.png</normaloff>:/icon/res/refresh.png</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QPushButton" name="btn_connect">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>连接</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>已连接设备</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lb_name">
            <property name="text">
             <string>名称:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lb_addr">
            <property name="text">
             <string>地址:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" rowspan="5">
          <widget class="QTextEdit" name="recv_textEdit"/>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QCheckBox" name="cb_hex_display">
           <property name="text">
            <string>HEX显示</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QCheckBox" name="cb_time">
           <property name="text">
            <string>加时间戳</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>编码:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="cb_encode">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>UTF-8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GBK</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QPushButton" name="btn_clear">
           <property name="text">
            <string>清空接收</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" rowspan="4">
          <widget class="QTextEdit" name="send_textEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="cb_hex_send">
           <property name="text">
            <string>HEX发送</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cb_CRLF">
           <property name="text">
            <string>加回车换行</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btn_send">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
